### PR TITLE
Fix/remove search api references

### PR DESCRIPTION
**Pull request by KamenDimitrov97**
### What

Removed all references of dp-search-api that have been copied over when copying the SDK

### How to review

run make lint, audit, test-all 
check for incosistencies and typos 

### Who can review

Andrea, Phil

### CODE REVIEW

File: LICENSE
Comment (andrea-kyurchiev) [inline LICENSE:1]: Why was this deleted?

Comment (KamenDimitrov97) [reply LICENSE:1]: because there was another LICENSE.md added by ONS
